web/e2e/specs/flows/steps/ingestion.ts

```diff
@@ -107,6 +113,8 @@ export default function (qaProjectDir) {
       await manageFlowPage.createFlow(flowData.flow1);
       await editFlowPage.addStep(flowData.flow1, json);
       await browser.sleep(5000);
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("json");
+      await expect(ingestStepPage.mlcpCommand.getText()).toContain("-input_file_type \"documents\"");
       await editFlowPage.clickRunFlowButton();
```

Comment:
Previous test in line 36 already verifies this. Does it add value to have these expects here.  Other duplicates are line 138,139 and 256,257.
Line 160 is fine since the file type is different.

Reply:
Fixed.